server/routes/induction/common/affectAbilityToWorkController.ts

```diff
@@ -21,6 +21,12 @@ export default abstract class AffectAbilityToWorkController extends InductionCon
     const affectAbilityToWorkForm = req.session.affectAbilityToWorkForm || toAffectAbilityToWorkForm(inductionDto)
     req.session.affectAbilityToWorkForm = undefined
 
+    // Check if we are in the midst of changing the main induction question set (in this case from short route to long route)
```

Comment:
All of the controllers in the `common` folder are abstract superclass controllers, essentially with the functionality for the "GET" endpoints. The reason we have them as superclasses like this is that we currently extend them with controllers for the Update functionality (which we are doing at the moment)
But soon we will look at the "Create" journey functionality, and will create new controllers for the "create" functionality. But the "GET" endpoints for the create journey will be the same, hence these abstract superclasses.

Anyway, the changes in these `common` controllers is to add handling for the Back links (the Back links are different depending on how you got to a given page - either direct from a Change link, or as part of a journey).
To manage the back link locations we have a concept of a 'page history' that we hold in session. 
The code we are adding to these controllers is to detect if we are part of an "update journey" and if so add the current page to the page history.